NBitcoin.Tests/Helpers/PrimitiveUtils.cs

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
This class contains a method used to to be in `transaction_test` .
Some functions (e.g. `RandomCoin` ) is quite reusable so I've moved it to here.